### PR TITLE
Added contibutedby feature and a blog - usage of monorepo

### PR DESCRIPTION
**Pull request by anuragts**

### CODE REVIEW

**Comment by anuragts** [comment]
Linked issue [#6 ]

File: src/layouts/BlogLayout.astro [hunk 1]
```diff
@@ -87,6 +87,7 @@ const classBody =
     <main class="mx-auto max-w-4xl md:px-20 p-8">
       <h1 class="sm:text-4xl text-3xl font-bold my-5">{frontmatter.title}</h1>
       <p class="dark:text-zinc-300 text-zinc-700 text-base">{pubDate}</p>
+        <span class="text-lg">By - {frontmatter.contributedBy}</span>
```
Comment (santoshyadavdev) [inline src/layouts/BlogLayout.astro]: Let's have it @twitteehandle

File: src/pages/tips/usage-of-monorepo.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (santoshyadavdev) [inline src/pages/tips/usage-of-monorepo.mdx]: I am not sure if mono-repo comes under GitHub tips, we want to have tips around using GitHub. We can get the contributedBy fix as of now.

**Review by santoshyadavdev — CHANGES_REQUESTED** [review]

File: src/pages/tips/usage-of-monorepo.mdx (hunk 2 above)
Comment (anuragts) [reply src/pages/tips/usage-of-monorepo.mdx]: Delete the monorepo blog, will add some tips soon.

**Comment by santoshyadavdev** [comment]
Thank you 💙

**Review by santoshyadavdev — APPROVED** [review]

**Pull request merged**